src/include/daos_srv/ras.h

```diff
@@ -81,29 +103,43 @@ ras_event_sev_enum_to_name(enum ras_event_sev severity)
 	case RAS_SEV_ERROR:
 		return "ERROR";
 	case RAS_SEV_INFO:
+	default:
 		return "INFO";
 	}
-
-	return RAS_SEV_UNKNOWN_STR;
 }
 
-enum ras_event_type {
-	/* ANY is a special case to match all types */
-	RAS_TYPE_ANY	= 0,
-	RAS_TYPE_RANK_STATE_CHANGE,
-	RAS_TYPE_INFO_ONLY,
-};
+/**
+ * Raise a RAS event and forward to the control-plane.
+ *
+ * \param[in] id	Unique event identifier.
+ * \param[in] msg	Human readable message.
+ * \param[in] type	Event type.
+ * \param[in] sev	Event instance severity.
+ * \param[in] hid	(Optional) Hardware component involved.
+ * \param[in] rank	(Optional) DAOS rank involved.
+ * \param[in] jid	(Optional) Client job involved.
+ * \param[in] puuid	(Optional) DAOS pool involved.
+ * \param[in] cuuid	(Optional) DAOS container involved.
+ * \param[in] oid	(Optional) DAOS object involved.
+ * \param[in] cop	(Optional) Recommended automatic control operation.
```

Comment:
I've been to-ing and fro-ing with the naming of these parameters, @johannlombardi requesting very short variable names and @mjmac requesting more verbose variable names, I have gone with a trade-off having short variable names on the ras.h structure and slightly more verbose in the proto & go structures. Rather than generating more churn now I'd like to keep as is and revisit with more consistent naming in the next iteration.

Reply:
OK with me

Reply:
I'm going to make this change as suggested in follow-up: https://jira.hpdd.intel.com/browse/DAOS-6436